docs/mkdocs.yml

```diff
@@ -37,6 +37,7 @@ pages:
   - Define a Model:
     - Using Sequential API: ProgrammingGuide/Model/Sequential.md
     - Using Functional API: ProgrammingGuide/Model/Functional.md
+    - Implement customized serializer: ProgrammingGuide/Model/Serialization.md
```

Comment:
I don't think we should put the link here. It has nothing to do with how you define a model. I think we should put it together with ```add-layer-criterion```. Maybe create a group named "Define Customized Layer/Criterion" to hold them.